core/utils/utils.go

```diff
@@ -437,6 +437,14 @@ func MustHash(in string) common.Hash {
 	return common.BytesToHash(out)
 }
 
+func MustHexDecodeString(s string) []byte {
```

Comment:
nit: If this is only used in tests, I think `cltest` would be a better place for this. I try to avoid panicking in general, maybe during initialization. `cltest` is a bit controversial, as it pulls in just about every package, but it is where we tend to keep test helpers.